### PR TITLE
[Torch Dialect] add support for adaptive_avgpool_1d

### PR DESCRIPTION
**Pull request by JianzheXiao**
as tittle , add support to decompose from adaptive_avgpool_1d to avgpool_1d

### CODE REVIEW

**Comment by JianzheXiao** [comment]
@ramiro050  Could you please help me review this PR? Thank you so much!

File: lib/Conversion/TorchToStablehlo/Pooling.cpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ static Value createInitialValueForAtenPoolingOp(Operation *op, Type elementTy,
                                                 PatternRewriter &rewriter) {
   auto constType = RankedTensorType::get({}, elementTy);
   // Avg pooling
-  if (isa<AtenAvgPool1dOp, AtenAdaptiveAvgPool2dOp, AtenAvgPool2dOp, AtenCumsumOp>(op)) {
```
Comment (ramiro050) [inline lib/Conversion/TorchToStablehlo/Pooling.cpp:38]: Is there a reason for adding it here? Given the decomposition, support for `AtenAvgPool1dOp` should be all that is needed

**Review by ramiro050 — APPROVED** [review]
Just a small comment, but other than that LGTM

File: lib/Conversion/TorchToStablehlo/Pooling.cpp (hunk 1 above)
Comment (JianzheXiao) [reply lib/Conversion/TorchToStablehlo/Pooling.cpp]: You are right, I just removed it from the file

**Comment by qingyunqu** [comment]
@ashay Cloud you please help fix the ci failure (git checkout error)?

**Comment by ashay** [comment]
I think @powderluv might have to reset the runners to work around the error.

**Pull request merged**